specs/StatusBarMessage.md

```diff
@@ -0,0 +1,106 @@
+<!-- USAGE
+  * Fill in each of the sections (like Background) below
+  * Wrap code with `single line of code` or ```code block```
+  * Before submitting, delete all <!-- TEMPLATE marked comments in this file,
+    and the following quote banner:
+-->
+# Background
+The browser has a Status bar that displays text when hovering over a link, or performing some activity. Currently, 
+developers are able to opt in to disable showing the status bar through the browser 
+settings.
+
+Developers would also like to be able to opt in to intercept the messages which would 
+normally be displayed by the Status bar, and show it using thier own custom UI. 
+# Description
+We propose a new event for WebView2 that would allow developers to 
+listen for Status bar updates which are triggered by activity on the WebView, and then handle those updates however they want in their applications.
```

Comment:
If the page does a `window.status = "Hello";` - does that result in a StatusBarMessageChanged event? Can it be read by the `StatusBarMessage` property?

I'm guessing the answer is "Whether the WebView honors script-driven status bar changes is at the discretion of the WebView and is not covered here. What we do promise is that the StatusBarMessage property accurately reflects the message shown in the status bar (if enabled) or what would be shown in the status bar (if disabled)."